frame/bridge/ethereum/linear-relay/src/lib.rs

```diff
@@ -106,7 +106,7 @@ pub trait Config: frame_system::Config {
 
 	type Event: From<Event<Self>> + Into<<Self as frame_system::Config>::Event>;
 
-	type EthereumNetwork: Get<EthereumNetworkType>;
+	type TargetNetwork: Get<EthereumNetwork>;
```

Comment:
```suggestion
	type BridgedNetwork: Get<EthereumNetwork>;
```

`Target` might be confused with target chain, this ethereum network here is actually source chain.